### PR TITLE
Improve Documentation with new name of driver

### PR DESCRIPTION
**Pull request by Yagora**
Added the name of drive in the doc and a way to start the AppImage on debian and ubuntu.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -217,6 +217,9 @@ <h2>Useful information</h2>
 										<strong>For Fedora users,</strong> you may check that you have installed the library <a href="https://pkgs.org/search/?q=librtmp" target="_blank">librtmp</a>.
 									</p>
 
+									<p>
+										<strong>For Debian or Ubuntu users,</strong> you may set an environment variable <code>LIBVA_DRIVER_NAME=iHD</code> or start the AppImage like this <code>LIBVA_DRIVER_NAME=iHD ./Shadow.AppImage --no-sandbox</code>.
+									</p>
```
Comment (NicolasGuilloux) [inline docs/index.html:222]: Maybe add "for recent Intel GPU" because older ones should stick with `i965` driver

**Review by NicolasGuilloux — CHANGES_REQUESTED** [review]

**Comment by NicolasGuilloux** [comment]
Thanks bud' for the PR, I really appreciate it :)

File: docs/index.html (hunk 1 above)
Comment (Yagora) [reply docs/index.html:222]: It's better like this ?

**Comment by Yagora** [comment]
You really helped me 2 years ago so it's nothing

File: docs/index.html (hunk 1 above)
Comment (NicolasGuilloux) [reply docs/index.html:222]: Perfect :)

**Review by NicolasGuilloux — APPROVED** [review]

**Pull request merged**